### PR TITLE
Update Grafana Cloud tutorial for k6 OSS

### PR DESCRIPTION
**Pull request by ppcano**
To use the new extension: https://github.com/grafana/xk6-output-prometheus-remote

### CODE REVIEW

File: src/data/extensions/extensions.js [hunk 1]
```diff
@@ -238,7 +238,7 @@ const extensions = [
   },
   {
     name: 'xk6-influxdbv2',
-    description: 'A k6 extension output to influxdb',
+    description: 'Export results to influxdb',
     url: 'https://github.com/li-zhixin/xk6-influxdbv2',
```
Comment (yorugac) [inline src/data/extensions/extensions.js:242]: We actually just added an "official" InfluxDB v2 extension here: https://github.com/grafana/xk6-output-influxdb
Perhaps, this link should be updated too?

**Review by yorugac — CHANGES_REQUESTED** [review]
Found just a typo, the content LGTM :+1: + a small suggestion about Influx extension.

File: src/data/markdown/translated-guides/en/04 Results visualization/05 Grafana Cloud.md [hunk 2]
```diff
@@ -1,100 +1,66 @@
...
 
-> An example repository using Docker to run Telegraf is available on GitHub at [https://github.com/k6io/example-k6-to-grafana-cloud](https://github.com/k6io/example-k6-to-grafana-cloud). 
+xk6 will create the k6 binary in the local folder. 
+
+> To learn more about how to build custom k6 versions, check out [xk6](https://github.com/grafana/xk6). 
 
 ## Run the test
 
-When telegraf is running, you can execute your k6 test and configure k6 to output the metrics to the telegraf instance. Telegraf will forward the k6 metrics to your Grafana Cloud Prometheus.
+Now, pass the Username, API key, and Remote Write Endpoint of the Grafana Cloud Prometheus Configuration to the k6 binary built with the extension as follow:
```
Comment (yorugac) [inline src/data/markdown/translated-guides/en/04 Results visualization/05 Grafana Cloud.md:53]:
```suggestion
Now, pass the Username, API key, and Remote Write Endpoint of the Grafana Cloud Prometheus Configuration to the k6 binary built with the extension as follows:
```

File: src/data/extensions/extensions.js (hunk 1 above)
Comment (ppcano) [reply src/data/extensions/extensions.js]: Thanks! Fixed by https://github.com/grafana/k6-docs/pull/474/commits/66cf51d757aca302bdc7186516172aa22b0fa9d0

**Pull request merged**